auto_submit/lib/validations/required_check_runs.dart

```diff
@@ -0,0 +1,5 @@
+// Copyright 2022 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+const List<String> requiredCheckRuns = ['ci.yaml validation'];
```

Comment:
It's weird that the GitHub API doesn't surface what is considered a required check run. You may want to leave a comment here as I went digging in the API to confirm this.

Reply:
Haha yeah I spent a bit of time trying to see if I could get that information as well. I will definitely add a note.